tests/tables/Makefile.am

```diff
@@ -32,4 +32,5 @@ EXTRA_DIST =					\
 	squash_space_with_context_2.utb		\
 	squash_space_with_correct.utb		\
 	squash_space_with_repeated.utb		\
+	table-dev-nocross.ctb		\
```

Comment:
Again here it might make sense to chage the name to something along the lines of `maketable`